benchmarks/benchmark_run.sh

```diff
@@ -0,0 +1,168 @@
+        # server:
+        OMP_NUM_THREADS=$OMP_NUM_THREADS numactl -N 0 -m 0 -C 0-$(($OMP_NUM_THREADS-1)) llm_on_ray-serve --config_file llm_on_ray/inference/models/vllm/llama-2-7b-chat-hf-vllm.yaml --simple --max_concurrent_queries $VALUE_INF --vllm_max_num_seqs $vllm_bs
+        # client:
+        numactl -N 1 -m 1 python benchmarks/benchmark_serving.py --model-endpoint-base http://localhost:8000/llama-2-7b-chat-hf --model-name llama-2-7b-chat-hf --dataset ./dataset/ShareGPT_V3_unfiltered_cleaned_split.json --num-prompts $num_prompts --dataset-format ShareGPT --vllm-engine --simple >> $log_path_vllm
+    done
+    for llmonray_bs in ${bs}
+    do
+        echo "RUN LLMonRay"
+        echo "RUN bs ${llmonray_bs}"
+        echo "bs: ${llmonray_bs}" >> $log_path_llmonray
+        # server:
+        OMP_NUM_THREADS=$OMP_NUM_THREADS numactl -N 0 -m 0 -C 0-$(($OMP_NUM_THREADS-1)) llm_on_ray-serve --config_file llm_on_ray/inference/models/llama-2-7b-chat-hf.yaml --simple --max_concurrent_queries $llmonray_bs
+        # client:
+        numactl -N 1 -m 1 python benchmarks/benchmark_serving.py --model-endpoint-base http://localhost:8000/llama-2-7b-chat-hf --model-name llama-2-7b-chat-hf --dataset ./dataset/ShareGPT_V3_unfiltered_cleaned_split.json --num-prompts $num_prompts --dataset-format ShareGPT --simple >> $log_path_llmonray
+    done
+}
+
+latency_throughput(){
+    num_iter=${1}
+    concurrent_query_num=${2}
+    input_tokens_length=${3}
+    output_tokens_length=${4}
+    log_path=${5}
+    if [ -f $log_path ]; then
+        rm $log_path
+    fi
+
+    # # server
+    OMP_NUM_THREADS=$OMP_NUM_THREADS numactl -N 0 -m 0 -C 0-$(($OMP_NUM_THREADS-1)) llm_on_ray-serve --config_file llm_on_ray/inference/models/vllm/llama-2-7b-chat-hf-vllm.yaml --simple --max_concurrent_queries $VALUE_INF --vllm_max_num_seqs $VALUE_INF
+
+    # client
+    for i in $(seq 1 $num_iter)
+    do
+        echo "Run iter $i"
+        echo "iter: $i" >> $log_path
+        for num_prompts in ${concurrent_query_num}
+        do
+            echo "Run num_prompts ${num_prompts}"
+            echo "num_prompts: ${num_prompts}" >> $log_path
+            numactl -N 1 -m 1 python benchmarks/benchmark_serving.py --model-endpoint-base http://localhost:8000/llama-2-7b-chat-hf --model-name llama-2-7b-chat-hf --dataset ./dataset/prompt.json --num-prompts $num_prompts  --dataset-format IPEX --input-tokens $input_tokens_length --max-new-tokens $output_tokens_length --track-token-latency --simple >> $log_path
+        done
+    done
+}
+
+get_best_latency(){
+    num_iter=${1}
+    input_tokens_length_li=${2}
+    output_tokens_length=${3}
+    log_path=${4}
+    if [ -f $log_path ]; then
+        rm $log_path
+    fi
+
+    # server
+    OMP_NUM_THREADS=$OMP_NUM_THREADS numactl -N 0 -m 0 -C 0-$(($OMP_NUM_THREADS-1)) llm_on_ray-serve --config_file llm_on_ray/inference/models/vllm/llama-2-7b-chat-hf-vllm.yaml --simple --max_concurrent_queries $VALUE_INF --vllm_max_num_seqs $VALUE_INF
```

Comment:
No need to define extra OMP_NUM_THREADS as we already supported this argument inside the code. This make an impression that user need to define the env before use.